rust/load/src/workloads.rs

```diff
@@ -36,5 +36,55 @@ pub fn all_workloads() -> HashMap<String, Workload> {
                 document: None,
             }),
         ),
+        (
```

Comment:
For later, but I am a little hesisitant about hard-coding the 30/70 part here. In general, I feel like workloads should be parameterized so that these values can be driven by runtime config.

Reply:
The hard coded piece is the default pattern.  You are free to shove json into the workload and make any pattern you wish.